src/typescript/azure-openapi-validator/rules/PathParameterMustNotBeEmpty.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { MergeStates, OpenApiTypes, rules } from '../rule';
+export const PathParametersMustNotBeEmpty: string = "PathParametersMustNotBeEmpty";
+
+rules.push({
+  id: "R2067",
+  name: PathParametersMustNotBeEmpty,
+  severity: "warning",
+  category: "RPCViolation",
+  mergeState: MergeStates.individual,
+  openapiType: OpenApiTypes.arm,
+
+  appliesTo_JsonQuery: "$.paths..parameters[?(@.in=='path')]",
+  run: function* (doc, node, path) {
+
+    const pathNodes: string[] = (<string>path[1]).split('/');
+
+    if (!Object.keys(node).includes("minLength") || node.minLength < 1) {
```

Comment:
JavaScript magic: this is equivalent to the (smaller) `!("minLength" in node)` or maybe even better:
``` JavaScript
typeof node.minLength !== "number"
```
That last thing checks the same as you do, but also ensures the type, so the comparison you make afterwards will not be between an apple and a number